### PR TITLE
Vendor `MCM v0.48.0`, Update golang version to 1.19.5

### PR DESCRIPTION
**Pull request by rishabh-11**
**What this PR does / why we need it**:
Vendor `MCM v0.48.0`. Update golang version to 1.19.5

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```other dependency
The following dependency is updated:
- github.com/gardener/machine-controller-manager v0.47.0 -> 0.48.0
```
```other operator
update golang version to 1.19.5
```

### CODE REVIEW

File: pkg/aws/aws_suite_test.go [hunk 1]
```diff
@@ -53,14 +55,11 @@ func newMachine(
...
 
-	// Don't initialize providerID and node if setMachineIndex == -1
+	// Don't initialize providerID if setMachineIndex == -1
 	if setMachineIndex != -1 {
 		machine.Spec = v1alpha1.MachineSpec{
 			ProviderID: fmt.Sprintf("aws:///eu-west-1/i-0123456789-%d", setMachineIndex),
 		}
-		machine.Status = v1alpha1.MachineStatus{
-			Node: fmt.Sprintf("ip-%d", setMachineIndex),
-		}
```
Comment (himanshu-kun) [inline pkg/aws/aws_suite_test.go:63]: we should replace this with `setting the node label`

**Review by himanshu-kun — CHANGES_REQUESTED** [review]
a minor change requested. rest looks good

File: pkg/aws/aws_suite_test.go (hunk 1 above)
Comment (rishabh-11) [reply pkg/aws/aws_suite_test.go]: done

**Comment by elankath** [comment]
Shouldn't this be something like:
```go
      machine.Labels = map[string]string{
                 v1alpha1.NodeLabelKey: fmt.Sprintf("ip-%d", setMachineIndex),
      }
```

**Comment by rishabh-11** [comment]
> Shouldn't this be something like:
> 
> ```go
>       machine.Labels = map[string]string{
>                  v1alpha1.NodeLabelKey: fmt.Sprintf("ip-%d", setMachineIndex),
>       }
> ```

yes it should be. i have changed it now

**Review by himanshu-kun — APPROVED** [review]
/lgtm

**Pull request merged**